system-addon/content-src/components/Base/Base.jsx

```diff
@@ -49,6 +49,7 @@ class Base extends React.Component {
         <div className="outer-wrapper">
           <main>
             {prefs.showSearch && <Search />}
+            <ManualMigration />
```

Comment:
Or maybe even match the `show` pattern with `showMigration` and have it start `true` then goes `false` when expired.